lessons/textstats.md

```diff
@@ -0,0 +1,597 @@
+
+
+    %matplotlib inline
+
+```
```

Comment:
Embarrassing to have left those things in there. 

abashedly, 

john

> On Sep 22, 2015, at 8:08 PM, W. Caleb McDaniel <notifications@github.com> wrote:
> 
> In lessons/textstats.md:
> 
> > @@ -0,0 +1,597 @@
> > +
> > +
> > +    %matplotlib inline
> > +
> > +```
> 
> @johnlaudun You'll need to remove these lines of "backticks" from around your YAML block, so that our site can pick up the relevant metadata. If you make this change on your fork, it should automatically update your pull request. Also, I presume the line that says matplotlib inline, and the notes to yourself about Markdown syntax, were inadvertently left in the file; please remove them to.
> 
> —
> Reply to this email directly or view it on GitHub.
>